agent/instrumentation/methods/README.md

```diff
@@ -0,0 +1,3 @@
+This is a copy of the upstream methods instrumentation module, but changed to emit SERVER spans
```

Comment:
can we add this note to MethodSingletons class? it used `SpanKindExtractor.alwaysServer()`. and then delete this readme?

Reply:
sure, ptal